test/integration/backward_compatible/parallel/nearcache/MigratedDataTest.js

```diff
@@ -42,21 +42,22 @@ describe('MigratedDataTest', function () {
                 deferred.reject(new Error('Partition table is not received!'));
             }
         }
-        checkPartitionTable(10);
+        checkPartitionTable(600);
         return deferred.promise;
     }
 
     async function waitUntilPartitionMovesTo(partitionService, partitionId, uuid) {
         const deferred = deferredPromise();
-        (function resolveOrTimeout(remainingTries) {
+        function resolveOrTimeout(remainingTries) {
```

Comment:
Minor: maybe as a simplification, we can get rid of the recursive function calls and replace it with a simple for loop. The same holds true for `waitForPartitionTableEvent`

Reply:
We can do that but I did not want to spend time on these